### ISSUE TITLE
最新版0.8.1的问题

### ISSUE THREAD

**Issue by GraceKafuu** [PanJinquan/pytorch-base-trainer#4]
1.不断打印torch版本号

INFO: {'lr': 0.01, 'train': {'Accuracy': 98.828125, 'loss': 0.034419890493154526}}
Train Epoch #4: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:42<00:00,  8.53s/it]
Train Epoch #4: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:41<00:00,  8.37s/it]
Train Epoch #4: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:42<00:00,  8.54s/it]
Train Epoch #4: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:43<00:00,  8.62s/it]
Test Epoch #4:   0%|                                                                                                    

torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0
torch version:1.12.0

2.感觉训练速度没有旧版快

**Comment by GraceKafuu** [comment]
3.还有一个严重问题，训练到第31epoch就终止了训练
